src/dice/ffi/metall.h

```diff
@@ -5,11 +5,13 @@
 #include <stddef.h>
```

Comment:
Possibly unused #include directive
Clang-Tidy: Inclusion of deprecated C++ header 'stddef. h'; consider using 'cstddef' instead 

also, its use is deprecated

Reply:
This is a C header, so not deprecated